### PR TITLE
Do not search for binary in current directory

### PR DESCRIPTION
**Pull request by DanielRuf**
Closes https://github.com/emscripten-core/emscripten/issues/8333

### CODE REVIEW

File: site/source/docs/api_reference/advanced-apis.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ While it is possible to edit **settings.js** manually, this is *highly discourag
 
 The small number of options that developers may have cause to change should be modified when the *emcc* tool is invoked. For example, ``EXPORTED_FUNCTIONS``: ::
 
-  ./emcc tests/hello_function.cpp -o function.html -s EXPORTED_FUNCTIONS="['_int_sqrt']"
+  emcc tests/hello_function.cpp -o function.html -s EXPORTED_FUNCTIONS="['_int_sqrt']"
```
Comment (sbc100) [inline site/source/docs/api_reference/advanced-apis.rst]: I think in these cases where you are expect to run it from the current working directoy its ok to keep it.  (anywhere where the input file is inside the emscripten repo)

File: site/source/docs/building_from_source/verify_emscripten_environment.rst [hunk 2]
```diff
@@ -62,7 +62,7 @@ Emscripten has a comprehensive test suite which may be used to further validate
 Troubleshooting
 ===============
 
-First run ``./emcc -v`` and examine the output to find missing components. You can also try ``./emcc --clear-cache`` to empty the :ref:`compiler's internal cache <emcc-clear-cache>` and reset it to a known good state.
+First run ``emcc -v`` and examine the output to find missing components. You can also try ``emcc --clear-cache`` to empty the :ref:`compiler's internal cache <emcc-clear-cache>` and reset it to a known good state.
```
Comment (sbc100) [inline site/source/docs/building_from_source/verify_emscripten_environment.rst]: Revert these too.

File: site/source/docs/tools_reference/emcc.rst [hunk 3]
```diff
@@ -14,7 +14,7 @@ Command line syntax
 
   emcc [options] file...
 
-(Note that you will need ``./emcc`` if you want to run emcc from your current directory.)
+(Note that you will need ``emcc`` if you want to run emcc from your current directory.)
```
Comment (sbc100) [inline site/source/docs/tools_reference/emcc.rst]: Certainly revert this one :)

File: site/source/docs/api_reference/advanced-apis.rst (hunk 1 above)
Comment (DanielRuf) [reply site/source/docs/api_reference/advanced-apis.rst]: Oops, a global search and replace is probably not the best solution. Pushed a new commit.

**Comment by kripken** [comment]
Please also add yourself to AUTHORS before we merge.

**Comment by DanielRuf** [comment]
Done.

**Comment by DanielRuf** [comment]
> Please also add yourself to AUTHORS before we merge.

Done.

**Comment by kripken** [comment]
Great, thanks @DanielRuf!

File: site/source/docs/building_from_source/verify_emscripten_environment.rst [hunk 4]
```diff
@@ -17,7 +17,7 @@ The first step in verifying the environment is to run Emscripten with the versio
 
 Open a terminal in the directory in which you installed Emscripten (on Windows open the :ref:`Emscripten Command Prompt <emcmdprompt>`). Then call the :ref:`Emscripten Compiler Frontend (emcc) <emccdoc>` as shown::
 
-  ./emcc -v
+  emcc -v
```
Comment (sbc100) [inline site/source/docs/building_from_source/verify_emscripten_environment.rst]: These ones should have the `./` in them as hey explicitly talk about opening a terminal in the directory where emscripten is installed..  without the `./` the emc cin the current working directory might not be the one being run.

There are quite a few cases like this.   I think its best to revert this change and try to re-land a narrower version.

**Comment by kripken** [comment]
Sorry, I thought this was lgtm'd. I'll revert.

**Comment by sbc100** [comment]
@DanielRuf sorry for the revert, we are still happy to accept a smaller version of this PR.

**Comment by DanielRuf** [comment]
Why revert and not push another commit? I will create another commit, squash them and open a new PR.

File: site/source/docs/building_from_source/verify_emscripten_environment.rst (hunk 4 above)
Comment (DanielRuf) [reply site/source/docs/building_from_source/verify_emscripten_environment.rst]: > There are quite a few cases like this. I think its best to revert this change and try to re-land a narrower version.

Mh =( Unfortunately I have no more time and motivation currently so I think I'll accept this and leave it as is - thanks for the support so far.

**Comment by sbc100** [comment]
You are right a fix forward would have worked too.   My thinking was that it would be easier to review this way.   A fix forward commit would not show the reviewer the whole story since it would only show the parts that got reverted, not the parts that were originally changed.

**Comment by kripken** [comment]
The reversion is just open, not landed - so we can still do it either way. I don't have a preference myself.

If you can fix this in a day or so, not reverting seems fine. If longer than that, then might be best to revert.

**Comment by kripken** [comment]
Sorry again for landing this too soon, and the confusion that caused.

@DanielRuf , did you decide what you prefer between landing the reversion (the reversion PR is still open - we have not actually reverted anything yet), or not? As we said, either option is fine.

**Comment by DanielRuf** [comment]
Hi, maybe I find the time to try all related steps in the changed parts and see if we have to change some.

**Comment by kripken** [comment]
It's been around a week, so probably we should resolve this. Since it's my fault for landing this too soon, I'll work on a followup PR to fix up the wrong parts here.
